src/style.css

```diff
@@ -0,0 +1,120 @@
+@import "./stars.css";
+
+p {
+  margin: 0;
+}
+
+.body {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+}
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px;
+}
+
+.card__code {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 16px 0;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 16px 16px 0;
+}
+
+.card__rating-reviews {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+/* #endregion */
+
+/* #region card__price */
+.card__price {
+  display: flex;
+  align-items: center;
+  height: 18px;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+}
+
+.card__price-text {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-sum {
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+/* #endregion */
+
+/* #region card__button */
+.card__button {
+  margin: 16px;
+  height: 40px;
+}
+
+.card__button-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 100%;
+  height: 100%;
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
```

Comment:
Please pay attention to your CSS properties order
https://9elements.com/css-rule-order/